cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/LineageHTTPHandler.java

```diff
@@ -115,6 +116,25 @@ public void datasetLineage(HttpRequest request, HttpResponder responder,
                          lineage, getCollapseTypes(collapse)), LineageRecord.class));
   }
 
+  /**
+   * Get the lineage information about a dataset. This method does not give any information on field level lineage.
+   *
+   * @param namespaceId the namespace in which the pipeline was created
+   * @param runId the program run ID
+   * @param stageName the stage name
+   */
+  @GET
+  @Path("/namespaces/{namespace-id}/runs/{run-id}/stagenames/{stage-name}/entityId")
+  public void normalizedFqn(HttpRequest request, HttpResponder responder,
+                             @PathParam("namespace-id") String namespaceId,
+                             @PathParam("run-id") String runId,
+                             @PathParam("stage-name") String stageName) throws Exception {
+    //accessEnforcer.enforce(datasetInstance, authenticationContext.getPrincipal(), StandardPermission.GET);
```

Comment:
add this for the output

Reply:
what does this comment mean? 

FYI, authorization checks should be handled within the admin class and not in the handler.